src/tools/run-make-support/src/rustc.rs

```diff
@@ -226,6 +226,18 @@ impl Rustc {
         self
     }
 
+    /// Add a directory to the library search path with a restriction. Equivalent to `-L KIND=PATH` in rustc.
```

Comment:
Suggestion: maybe specify that the restriction is the *kind* of search path? We could also link to <https://doc.rust-lang.org/rustc/command-line-arguments.html#-l-add-a-directory-to-the-library-search-path>.

Reply:
I can't add the link, because it's over 100 characters and tidy would complain, but I did add a precision.

```
    /// Add a directory to the library search path with a restriction, where `kind` is a dependency
    /// type. Equivalent to `-L KIND=PATH` in rustc.
```

Reply:
You can add `// ignore-tidy-linelength` to make tidy ignore this.